scripts/risk_pipeline/data_prep/get/reservoir_nft_data.py

```diff
@@ -0,0 +1,149 @@
+import requests
+import pandas as pd
+import argparse
+import time
+import os
+import sys
+sys.path.insert(0, os.getcwd()+'/scripts/risk_pipeline')
+from helpers import helpers  # noqa: E402
+
+FLOORS = "https://api.reservoir.tools/events/collections/floor-ask/v1"
+SALES = "https://api.reservoir.tools/sales/v4"
+
+
+def epoch_time(ts_string):
+    time_struct = time.strptime(ts_string, "%Y-%m-%d-%H-%M")
+
+    # Convert the time struct to Unix time and return
+    return int(time.mktime(time_struct))
+
+
+def get_params():
+    """
+    Get parameters from command line
+    """
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
```

Comment:
same here. Perhaps the arguments could be defined all in one place, as tuples, and then star expanded, like:

collection_name = ( '--collection_name', type=str, help='Name of the collection. Ex: BAYC')

then to add this you do parser.add_argument(*collection_name)